src/flags.rs

```diff
@@ -322,11 +324,11 @@ impl<'this> RustcCodegenFlags<'this> {
                 // https://doc.rust-lang.org/rustc/linker-plugin-lto.html
                 if self.linker_plugin_lto.unwrap_or(false) {
                     // https://clang.llvm.org/docs/ClangCommandLineReference.html#cmdoption-clang-flto
-                    let cc_val = match self.lto {
-                        Some("thin") => "thin",
-                        _ => "full",
-                    };
-                    push_if_supported(format!("-flto={cc_val}").into());
+                    // It has to be thin LTO because llvm linker plugin/lld uses thin LTO by default.
+                    // And for thin LTO to work, the archive also has to be compiled using thin LTO,
+                    // since thin LTO generates extra information that fat LTO does not generate that
+                    // is required for thin LTO process.
```

Comment:
Six "thin LTO" in two sentences. Well, not that I care that much, but it does sting the eye. It also feels that it would look worse in isolation, i.e. not in the diff context. I mean
```
- push_if_supported(format!("-flto={cc_val}").into());
+ // It has to be thin LTO because llvm linker plugin/lld uses thin LTO by default.
```
looks like meaningful "rebuttal," while
```
// https://clang.llvm.org/docs/ClangCommandLineReference.html#cmdoption-clang-flto
// It has to be thin LTO because llvm linker plugin/lld uses thin LTO by default.
```
feels like "wow, where does it come from?"

Reply:
Thanks updated

Reply:
Five "thin LTO"-s, only one down... But what does second sentence add now? It's basically the first sentence... Again, not that I care that much :-) Just do you :-)